sklearn/tree/_export.py

```diff
@@ -299,6 +299,9 @@ def node_to_str(self, tree, node_id, criterion):
         if self.impurity:
             if isinstance(criterion, _criterion.FriedmanMSE):
                 criterion = "friedman_mse"
+            elif (isinstance(criterion, _criterion.MSE)
+                  or criterion == "squared_error"):
+                criterion = "mse"
```

Comment:
I would prefer `squared_error` so there isn't any confusion. (Yes it would take up more space.)